src/components/CommentsCard.jsx

```diff
@@ -16,8 +21,29 @@ function CommentsCard({ article_id, username }) {
       .catch(() => {
         setIsError(true);
       });
+
+ 
   }, [comments]);
 
+  useEffect(()=>{
+
+      if(isDeleting){
+      deleteCommentByCommentId(commentId).then(()=>{
```

Comment:
This is fine, and nice that you're giving the user some feedback that the comment is in the process of being deleted, however to keep consistent with your post functionality (and make it a little less jumpy on the screen), optimistic rendering would be nice.